### PR TITLE
regenerating files after removing duplicates from HTML

### PR DESCRIPTION
**Pull request by econnally**
@karln per your slack request. hopefully this is a good way to manage requests like this LMK

https://app.shortcut.com/illustrative-mathematics/story/8321/remove-duplicate-fields-from-html

### CODE REVIEW

**Comment by econnally** [comment]
@karln i didn't notice before but it's using a different URL let me fix taht real quick

File: build/05-05/activity-node-244456.ocx [hunk 1]
```diff
@@ -50,7 +50,7 @@
   "encoding": [
     {
       "encodingFormat": "application/msword",
-      "contentUrl": "Grade6-2-2-3-Activity-student-task-statements.docx"
+      "contentUrl": "***TODO*** what URL should we use?"
```
Comment (econnally) [inline build/05-05/activity-node-244456.ocx:53]: @karln do you want me to kludge my around stuff like this? i can come up with a naming scheme for docx files based on your entries

Comment (karln) [reply build/05-05/activity-node-244456.ocx:53]: If it isn't too much trouble.

Comment (karln) [reply build/05-05/activity-node-244456.ocx:53]: Maybe I should merge the other two PRs before we merge this one? What do you think, @dylandechant ?

Comment (econnally) [reply build/05-05/activity-node-244456.ocx:53]: @karln yes then i can rebase

Comment (karln) [reply build/05-05/activity-node-244456.ocx:53]: okay, that's done.

**Review by karln — CHANGES_REQUESTED** [review]
Looks like this is going to break all the .docx links.

File: build/05-05/activity-node-244456.ocx (hunk 1 above)
Comment (econnally) [reply build/05-05/activity-node-244456.ocx]: ok @karln if this looks OK i can rebase/merge

**Review by karln — CHANGES_REQUESTED** [review]
Activities look good.

Assessments and cool downs are overwriting the .docx references.

And it looks like we're back to including the practice-problem-node files.

**Comment by econnally** [comment]
> Looks like this is going to break all the .docx links.

@karln what links are broken now

**Comment by karln** [comment]
@econnally that was an old comment, sorry.

**Comment by econnally** [comment]
> @econnally that was an old comment, sorry.

ok i don't know why pp nodes are there; will remove. are there other issues?

**Comment by econnally** [comment]
> ok i don't know why pp nodes are there; will remove. are there other issues?
@karln ^^

**Comment by karln** [comment]
@econnally The only other thing I'm seeing is that assessments and cool downs are overwriting the .docx references. Not sure how to resolve that.

**Comment by econnally** [comment]
@karln we WANT practice problem set nodes, but not prac prob nodes. correct?

**Comment by karln** [comment]
> @karln we WANT practice problem set nodes, but not prac prob nodes. correct?

That's correct. We want the sets, not the individual items.

**Comment by econnally** [comment]
> @econnally The only other thing I'm seeing is that assessments and cool downs are overwriting the .docx references. Not sure how to resolve that.

take a look now @karln @dylandechant

**Review by karln — APPROVED** [review]

**Comment by karln** [comment]
This looks pretty good. There's still a few issues around practice problems, but that's a different story so I'm going to accept this PR, fix the stuff that needs to be tweaked by hand, and we'll deliver this as an initial draft.

**Pull request merged**